Is the typescript definition exported correctly?

I'm having trouble with unfetch >= 3.1.1. The new TypeScript definition actually broke our build, and I'm not sure how to fix it.

The change was made in PR https://github.com/developit/unfetch/pull/89, in which the default export was removed from `src/index.d.ts`. Was this intentional? The PR is named "Fix _isomorphic-unfetch_ definition" but `packages/isomorphic-unfetch/index.d.ts` is not changed, instead `src/index.d.ts` was changed.

Since `src/index.mjs` export default, shouldn't `src/index.d.ts` also export default?

Same problem with `unfetch@3.0.0` (`webpack@3.12.0`, `ts-loader@2.3.7`, `typescript@2.9.2`):

```
error TS7016: Could not find a declaration file for module 'unfetch'. '/home/user/Edge/Demo/node_modules/unfetch/dist/unfetch.js' implicitly has an 'any' type.
```

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -14,4 +14,4 @@ declare namespace unfetch {
 
 declare const unfetch: typeof fetch;
 
-export = unfetch;
+export default unfetch;

